cypress/e2e/hub/namespaces.cy.ts

```diff
@@ -28,155 +25,309 @@ describe('Namespaces', () => {
+    cy.intercept('POST', hubAPI`/role_user_assignments/`).as('userRoleAssignment');
+    cy.createHubUser().then((hubUser) => {
+      cy.clickTab('User Access', true);
+      cy.getByDataCy('add-roles').click();
+      cy.verifyPageTitle('Add roles');
+      cy.getWizard().within(() => {
+        cy.contains('h1', 'Select user(s)').should('be.visible');
+        cy.selectTableRowByCheckbox('username', hubUser.username, { disableFilter: true });
+
+        cy.clickButton(/^Next/);
+        cy.contains('h1', 'Select roles to apply').should('be.visible');
+        cy.filterTableByTextFilter('name', 'galaxy.collection_namespace_owner', {
+          disableFilterSelection: true,
+        });
+        cy.selectTableRowByCheckbox('name', 'galaxy.collection_namespace_owner', {
+          disableFilter: true,
+        });
+        cy.filterTableByTextFilter('name', 'galaxy.collection_publisher', {
+          disableFilterSelection: true,
+        });
+        cy.selectTableRowByCheckbox('name', 'galaxy.collection_publisher', {
+          disableFilter: true,
+        });
+        cy.clickButton(/^Next/);
+        cy.contains('h1', 'Review').should('be.visible');
+        cy.verifyReviewStepWizardDetails('users', [hubUser.username], '1');
+        cy.verifyReviewStepWizardDetails(
+          'hubRoles',
+          [
+            'galaxy.collection_namespace_owner',
+            'Change and upload collections to namespaces.',
+            'Credential Admin',
+            'Upload and modify collections.',
+          ],
+          '2'
+        );
+        cy.clickButton(/^Finish/);
+        cy.wait('@userRoleAssignment')
+          .its('response')
+          .then((response) => {
+            expect(response?.statusCode).to.eql(201);
+          });
+      });
+      cy.getModal().within(() => {
+        cy.clickButton(/^Close$/);
+      });
+      cy.getModal().should('not.exist');
+      cy.verifyPageTitle(namespace.name);
+      cy.selectTableRowByCheckbox('username', hubUser.username, {
+        disableFilter: true,
+      });
+      removeRoleFromListRow('galaxy.collection_namespace_owner', 'user');
+      cy.selectTableRowByCheckbox('username', hubUser.username, {
+        disableFilter: true,
+      });
+      removeRoleFromListRow('galaxy.collection_publisher', 'user');
+    });
+  });
+
+  it('create a new namespace, from the team access tab assign a user and apply role(s) to the team of the namespace', () => {
+    cy.intercept('POST', hubAPI`/role_team_assignments/`).as('teamRoleAssignment');
+    cy.createHubTeam().then((hubTeam) => {
+      cy.clickTab('Team Access', true);
+      cy.getByDataCy('add-roles').click();
+      cy.verifyPageTitle('Add roles');
+      cy.getWizard().within(() => {
+        cy.contains('h1', 'Select user(s)').should('be.visible');
+        cy.selectTableRowByCheckbox('username', hubTeam.name, { disableFilter: true });
```

Comment:
Since this is the teams table, I think the first param should be `name` instead of `username`